src/Categories/Object/Subobject/Properties.agda

```diff
@@ -44,7 +43,10 @@ module _ {o ℓ e} {𝒞 : Category o ℓ e} (has-pullbacks : ∀ {A B X} → (f
     { F₀ = Subobjects 𝒞
     ; F₁ = λ f → record
       { fun = morphism f
-      ; monotone = λ {(α , m) (β , n)} h → monotone f {(α , m)} {β , n} h
+      ; isOrderHomomorphism = record
+        { cong = {!!}
```

Comment:
You can derive the proof for this from monotonicity and antisymmetry of the target/codomain poset. No need for a more complicated construction.

Reply:
Thanks!